src/borg/archiver/diff_cmd.py

```diff
@@ -19,10 +20,12 @@ class DiffMixIn:
     @with_archive
     def do_diff(self, args, repository, manifest, archive):
         """Diff contents of two archives"""
+        if args.format is not None:
+            format = args.format
         if args.content_only:
             format = "{content}{link}{directory}{blkdev}{chrdev}{fifo} {path}{NL}"
         else:
-            format = args.format or "{change} {path}{NL}"
+            format = os.environ.get("BORG_DIFF_FORMAT", "{change} {path}{NL}")
```

Comment:
That looks strange. How can options be combined? else if?

Reply:
Oops, I forget it 😵